src/containers/PrivacyPolicy/index.tsx

```diff
@@ -0,0 +1,341 @@
+import React, {FC} from 'react';
+
+import {A, PageTitle} from 'components';
+
+import './PrivacyPolicy.scss';
+
+const PrivacyPolicy: FC = () => {
+  return (
+    <>
+      <PageTitle title="Privacy Policy" />
+      <div className="PrivacyPolicy">
+        <p>Last Updated: April 5, 2021</p>
+        <h1 className="PrivacyPolicy__title">Privacy Policy</h1>
+        <h2 className="PrivacyPolicy__subtitle">Introduction</h2>
+        <p>
+          This privacy policy (“Policy”) describes how thenewboston in California (“Company,” “we,” and “our”)
```

Comment:
I think that "thenewboston in California" is not the best way to refer to the project. If we target compliance to California laws this must be stated specifically, otherwise, I am not sure if thenewboston has a location as of now.

Also change "we," to "we"